src/v2/Apps/Fair/Routes/FairArtworks.tsx

```diff
@@ -45,7 +44,7 @@ const FairArtworksFilter: React.FC<FairArtworksFilterProps> = props => {
   // For some reason, they are undefined when `useSystemContext()` is referenced
   // in <ArtistsFilter />. So, pass as props for now.
   const Filters = () => (
-    <Box pr={2}>
```

Comment:
This goes away because I'm just using `GridColumns` in the overall page layout. (Removing this also fixes a long standing bug where there was the 20px right padding on mobile views which have full-width filters)